src/events/evaluation.js

```diff
@@ -1,15 +1,27 @@
-function addEventstoEval(){
+function addEventsToEval() {
     const newEvalBtn = document.getElementById('newEvaluationBtn');
     const logoutBtn = document.getElementById('logoutBtn');
-    newEvalBtn.addEventListener("click",function(e){
-        event.preventDefault();
-        theBody.innerHTML ='';
-        theBody.innerHTML = NewEvaluationForm();
-        addEventstoNewEval();
+    const container = document.getElementById('app');
+    const detailsBtn = document.getElementsByClassName("detailsBtn");
+
+    newEvalBtn.addEventListener("click", function (e) {
+        e.preventDefault();
+        container.innerHTML = NewEvaluationForm();
+        addEventsToNewEval();
+        addPlaceHolder();
     })
-    logoutBtn.addEventListener("click",function(e){
-        event.preventDefault();
-        theBody.innerHTML = LoginPage();
+
+    logoutBtn.addEventListener("click", function (e) {
+        e.preventDefault();
+        container.innerHTML = LoginPage();
         addEventstoLogin();
     })
-}
\ No newline at end of file
+
+    detailsBtn.addEventListener("click", function (e) {
```

Comment:
Does this work the way you expect it to work or does it throw an error at runtime? Please note that some DOM traversal APIs return collections of HTML elements, as opposed to singular HTML elements, which are the event dispatchers you seem to be trying to leverage here!